Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -224,10 +237,39 @@ def examineFiles(self, files):
             for tag in subseriesTags:
                 value = slicer.dicomDatabase.fileValue(file, self.tags[tag])
                 value = value.replace(",", "_")  # remove commas so it can be used as an index
+
                 if tag not in subseriesValues:
                     subseriesValues[tag] = []
-                if not subseriesValues[tag].__contains__(value):
+
+                if tag in vectorTags:
+                    if value != "":
+                        directionCosine = [float(i) for i in value.split("\\")]
+                        directionCosineA = numpy.array(directionCosine[:3])
+                        directionCosineB = numpy.array(directionCosine[-3:])
+
+                        found = False
+                        for subseriesValue in subseriesValues[tag]:
+                            subserDirCos = [float(i) for i in subseriesValue.split("\\")]
+
+                            subserDirCosA = numpy.array(subserDirCos[:3])
+                            # eps value default to 1.e-6 as same behaviour of ITK numeric comparison logic.
+                            # rtol=0, atol=eps
+                            if not numpy.allclose(directionCosineA, subserDirCosA, 0., eps):
+                                continue
+
+                            subserDirCosB = numpy.array(subserDirCos[-3:])
+                            # eps value default to 1.e-6 as same behaviour of ITK numeric comparison logic.
```

Comment:
I don't think you need to repeat this comment

Reply:
I agree. Specify `absoluteTolerance` and `relativeTolerance` variables somewhere above, refer to the ITK implementation there (add least className::methodName, maybe also a full URL).

Reply:
done in https://github.com/Slicer/Slicer/pull/7362/commits/1ccb8ecb92cfb2c2a8db4d2cf5cdc46da5a2c04d